app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
             setupDrawerContent(navigationView);
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary));
```

Comment:
I believe setting this in the theme palette as colorPrimary etc. Results in less code and more separation.

```xml
<style name="Theme.MyTheme" parent="Theme.AppCompat.Light">
    <!-- colorPrimary is used for the default action bar background -->
    <item name="colorPrimary">@color/my_awesome_color</item>

    <!-- colorPrimaryDark is used for the status bar -->
    <item name="colorPrimaryDark">@color/my_awesome_darker_color</item>

    <!-- colorAccent is used as the default value for colorControlActivated
         which is used to tint widgets -->
    <item name="colorAccent">@color/accent</item>

    <!-- You can also set colorControlNormal, colorControlActivated
         colorControlHighlight & colorSwitchThumbNormal. -->
</style>
```
Please refer to https://chris.banes.me/2014/10/17/appcompat-v21/